app/uk/gov/hmrc/tai/connectors/JourneyCacheConnector.scala

```diff
@@ -84,4 +84,8 @@ class JourneyCacheConnector @Inject()(httpHandler: HttpHandler, servicesConfig:
   def flushWithEmpId(journeyName: String, empId: Int)(implicit hc: HeaderCarrier): Future[TaiResponse] =
     httpHandler.deleteFromApi(cacheUrl(s"$journeyName/$empId")).map(_ => TaiSuccessResponse)
 
+  def testOnlyCacheUrl(journeyName: String) = s"$serviceUrl/tai/test-only/journey-cache/$journeyName"
```

Comment:
why do we need to make a new route, why cant we just use the existing method for flushing the cache?

Reply:
This is just to keep the test ones separate